### ISSUE TITLE
RISC-V Toolchain installation & download

### ISSUE THREAD

**Issue by pedro-javierf** [Obijuan/RISC-V-FPGA#4]
What is the required RISC-V toolchain for this project and where can it be downloaded/Installed? Is this standard riscv-gnu-toolchain or should be something else?

`riscv32-unknown-elf-gcc`
`riscv32-unknown-elf-objcopy`
etc

Thanks,

**Comment by Obijuan** [comment]
Yes, that is the standard toolchain. You can also use the Rars assembler/simulator if you like, for generating the executable file. I still have not documented how to do it, but I will

**Comment by pedro-javierf** [comment]
For anyone interested in getting the demo firmware to work as fast as possible, it is possible to use the following pre-built riscv32-gnu toolchain here: https://github.com/stnolting/riscv-gcc-prebuilt (the [rv32i-2.0.0](https://github.com/stnolting/riscv-gcc-prebuilt/releases/tag/rv32i-2.0.0) version) to generate compatible binaries that are compatible for the RISC-V core in the Alhambra II. I've only tested in Ubuntu 20 VM.

The build instructions for the 32-bit gnu toolchain are not always up-to-date so this can be handy.
